packages/backend/src/api/passwordResetLinksRouter.ts

```diff
@@ -0,0 +1,44 @@
+import express from 'express';
+import { userService } from '../services/services';
+import { unauthorisedInDemo } from './authentication';
+
+export const passwordResetLinksRouter = express.Router();
+
+passwordResetLinksRouter.get(
+    '/:code',
+    unauthorisedInDemo,
+    async (req, res, next) =>
+        userService
+            .getPasswordResetLink(req.params.code)
+            .then(() => {
+                res.json({
+                    status: 'ok',
+                });
+            })
+            .catch(next),
+);
+
+passwordResetLinksRouter.post('/', unauthorisedInDemo, async (req, res, next) =>
+    userService
+        .recoverPassword(req.body)
+        .then(() => {
+            res.json({
+                status: 'ok',
+            });
+        })
+        .catch(next),
+);
+
+passwordResetLinksRouter.patch(
```

Comment:
Why is this a patch? imo it would be clearer to have different routes for recover vs. reset (I keep mixing those up). e.g. POST /user/password with the invite code and new password. And leave POST /password-reset as the way to initiate a reset

Reply:
We already have `POST user/password` to edit the user password when it is authenticated. 
What you think about changing the reset endpoint to `PATCH user/password` ?

Reply:
Ah I didn't see the existing endpoint on `/user/password`! 

Patch is usually for "partially update this single resource"

Perhaps leave `POST /user/password` to update the password (though `PUT` would be nice :) )
And then move password reset to `POST /user/password/reset` ?